internal/api/httpsrv/handler_issuer.go

```diff
@@ -81,12 +97,23 @@ func (h *apiHandler) GetIssuerByID(ctx context.Context, req GetIssuerByIDRequest
 }
 
 func (h *apiHandler) UpdateIssuer(ctx context.Context, req UpdateIssuerRequestObject) (UpdateIssuerResponseObject, error) {
+	// We must fetch the issuer to retrieve the owner so we may check for permission to update.
+	iss, err := h.engine.GetIssuerByID(ctx, req.Id)
+	switch err {
+	case nil:
+	case types.ErrorIssuerNotFound:
```

Comment:
Using `errors.Is` here is more forward-compatible, though maybe we don't need to do that here.

Reply:
indeed. I copied this from the GetIssuerByID method above. I, so keeping consistency, but I can update these if preferred.

Reply:
Nah, consistency is better - let's leave it as-is for now then.